content/blog/2024/04/2024-04-12-election-announcement.md

```diff
@@ -0,0 +1,102 @@
++++
+date = "2024-04-12"
+title = "Announcing the 2024 Governing Board elections"
+path = "/blog/2024/04/12/election-announcement"
```

Comment:
```suggestion
```

No need for the `path` by default, it's only useful for TWIMs becasue Zola can't handle clashes :)